src/a11y-demo/BUILD.bazel

```diff
@@ -21,5 +22,5 @@ ng_module(
         "@npm//@angular/animations",
         "@npm//@angular/forms",
         "@npm//@angular/router",
-    ] + CDK_TARGETS + MATERIAL_TARGETS,
+    ] + CDK_TARGETS + MATERIAL_NO_TEST_TARGETS,
```

Comment:
An option would be, instead of creating two exports, export an object that has `ALL_TARGETS` and `WITHOUT_TEST_TARGETS`
e.g. `MATERIAL.ALL_TARGETS` and `MATERIAL.WITHOUT_TEST_TARGETS`

Reply:
Sounds like an option, but I'm not sure if we benefit from it. Especially since it would be not consistent with the `SCSS`export variable for example.

Regardless of whether we use one or two exports, I guess we'll always use something like `NO_TESTS` or `WITHOUT_TESTS`. And this is the part where I look for an alternative 😞 (it feels somehow dirty but it's actually the most clear name; and will avoid confusion)

Reply:
I'd prefer something like `material_component_entrypoints` and `material_testing_entrypoints`, and anywhere you wanted both you _explicitly_ concat them, which I think would prevent anyone from accidentally including something they didn't mean to. 

(side note: I looked up the bzl style guide for this change, and they do recommend lower_snake_case, which we could do at a later date)

Reply:
I like the idea of just having non-testing and testing. Concatenating both should be straightforward enough 👍

Slightly unsure about `component` though. Not all entry-points expose a component. If we can improve that somehow, then I think we have a good naming convention. Can't think of anything else than `no_test`.

Reply:
`component` isn't perfect, but I think it's preferable to something like `without_tests`. I also considered words like `primary`, `main`, `root`, `top` `lib`, `browser`, but I didn't think of any of these work as well.

Reply:
just switched it to `no_test` for now. I'll handle the lower snake case in a follow-up (we have more places that need to be updated).